### PR TITLE
Gain control

### PR DESCRIPTION
**Pull request by ChristophTobler**
I added the capacitor for the Lidar Lite and used the telemetry port instead. Now I have less spikes.
I was able to fly with this PR indoors
https://www.youtube.com/watch?v=3ibqe5XjpeQ&list=UUkzTT-UBMp2rjuzckbBW3xw

### CODE REVIEW

File: src/mainloop.cpp [hunk 1]
```diff
@@ -54,8 +54,13 @@
 #define EXPOSURE_P_GAIN 100.0f
 #define EXPOSURE_I_GAIN 0.5f
 #define EXPOSURE_D_GAIN 0.5f
+#define GAIN_P_GAIN 50.0f
+#define GAIN_I_GAIN 0.5f
+#define GAIN_D_GAIN 0.5f
 #define EXPOSURE_CHANGE_THRESHOLD 30.0
 #define EXPOSURE_ABOSULUTE_MAX_VALUE 1727
+#define GAIN_CHANGE_THRESHOLD 15.0
+#define GAIN_ABOSULUTE_MAX_VALUE 100 //TODO check
```
Comment (zehortigoza) [inline src/mainloop.cpp]: You want to know the maximum gain that the camera supports? It is 127.

**Review by zehortigoza — COMMENTED** [review]
LGTM, I will just add again on top of OpticalFlow submodule this patch https://github.com/zehortigoza/OpticalFlow/commit/099566de18a54ddd824b641d7b43773e2a1741c0
Otherwise the integration with Yocto would require another receipt to install OpticalFlow lib.
Also make this the master branch.

Big thank you.

File: src/mainloop.cpp (hunk 1 above)
Comment (ChristophTobler) [reply src/mainloop.cpp]: fixed. Thanks!

**Comment by ChristophTobler** [comment]
> I will just add again on top of OpticalFlow submodule this patch zehortigoza/OpticalFlow@099566d

Sure :+1:

**Comment by zehortigoza** [comment]
Merged into master branch, thanks

**Pull request closed without merging**